applovin_max/android/src/main/java/com/applovin/applovin_max/AppLovinMAX.java

```diff
@@ -93,6 +93,8 @@ public class AppLovinMAX
     private List<String> targetingKeywordsToSet;
     private List<String> targetingInterestsToSet;
 
+    private final Map<String, String> extraParametersToSet = new HashMap<>();
```

Comment:
Can you move this up to just below `locationCollectionEnabledToSet`?   `extraParameter` belongs to General Public API, and we want to group them as much as possible.